src/WPF/WPF.Viewer/ApiKeyPrompt.xaml

```diff
@@ -7,10 +7,10 @@
         <TextBlock x:Name="Instructions"
                    Margin="5"
                    TextWrapping="Wrap">
-            Some content used in the samples requires a developer API key for access. Go to<Hyperlink NavigateUri="https://links.esri.com/arcgis-api-keys" RequestNavigate="Hyperlink_RequestNavigate">
+            Some content used in the samples requires a developer API key for access. Go to<Hyperlink NavigateUri="https://links.esri.com/create-an-api-key" RequestNavigate="Hyperlink_RequestNavigate">
```

Comment:
```suggestion
            Some content used in the samples requires a API key for access ArcGIS location services. Go to <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key" RequestNavigate="Hyperlink_RequestNavigate"> tutorial to create a new API Key.
```

Reply:
Since this is the samples you will need to advise them which privileges they must enable on the API Key. I think this is typically the basemaps, geocoding, and routing privileges.